pkgs/applications/science/astronomy/gildas/wrapper.patch

```diff
@@ -0,0 +1,18 @@
+diff --new-file -r -u gildas-src-feb17d.orig/admin/wrapper.sh gildas-src-feb17d/admin/wrapper.sh
```

Comment:
Is there a reason a new file is created completely from a patch instead of using the `makeWrapper` function?